qa/suites/fs/fscrypt/overrides/osd.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        debug bluestore: 20
```

Comment:
No objections from me, but do you have reason to believe that the debug message enhanced in https://github.com/ceph/ceph/pull/54571 wouldn't be sufficient for identifying whether the issue is on the OSD side or on the kernel client side?

Reply:
The `bluestore` code will have more detail about the extents. Just in case this is really a ceph side bug, no need to add or enable more debug logs to reproduce it twice in future.

Reply:
Fine with me.